src/components/IssueListItem.js

```diff
@@ -7,6 +7,10 @@ import {IssueOpenedIcon, CommentIcon, MilestoneIcon} from "@primer/octicons-reac
 
 dayjs.extend(relativeTime);
 
+function fontContrast(color) {
+  return Number("0x" + color) > 16000000 ? "black" : "white";
```

Comment:
@filiptronicek We need to figure out the correct threshold value `16000000` will not work for all light colors! We should try the values in trial and error method.

Reply:
I will try to look at any online resources, to get a better picture of what the treshold value should be

Reply:
Nonetheless, it looks it is doing a pretty good job at least in our repo
![image of labels](https://user-images.githubusercontent.com/29888641/86457585-2f676900-bd24-11ea-8e65-28080ad736bf.png)

Reply:
@filiptronicek Can you check with `chore` label?

Reply:
Sure thing!

Reply:
It looks like this one is off, it displays it in white, but should in black

Reply:
The `threshold` needs to be 13000000 or less for it to be dark